Description: Update Italian manual page
 Bring Italian manual page up-to-date.
Forwarded: Marco Peereboom <marco@peereboom.us>
Author: Andrea Bolognani <eof@kiyuko.org>
Last-Update: 2013-02-09
Index: spectrwm.git/spectrwm_it.1
===================================================================
--- spectrwm.git.orig/spectrwm_it.1	2013-02-03 21:47:02.144631643 +0100
+++ spectrwm.git/spectrwm_it.1	2013-02-09 14:51:19.477009268 +0100
@@ -1,5 +1,9 @@
-.\" Copyright (c) 2009 Marco Peereboom <marco@peereboom.us>
+.\" Copyright (c) 2009-2012 Marco Peereboom <marco@peereboom.us>
 .\" Copyright (c) 2009 Darrin Chandler <dwchandler@stilyagin.com>
+.\" Copyright (c) 2011-2012 Reginald Kennedy <rk@rejii.com>
+.\" Copyright (c) 2011-2012 Lawrence Teo <lteo@lteo.net>
+.\" Copyright (c) 2011-2012 Tiago Cunha <tcunha@gmx.com>
+.\" Copyright (c) 2012 David Hill <dhill@mindcry.org>
 .\"
 .\" Permission to use, copy, modify, and distribute this software for any
 .\" purpose with or without fee is hereby granted, provided that the above
@@ -13,7 +17,7 @@
 .\" ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
 .\" OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\"
-.Dd $Mdocdate: September 15 2011 $
+.Dd $Mdocdate: February 15 2012 $
 .Dt SPECTRWM 1
 .Os
 .Sh NOME
@@ -23,10 +27,10 @@
 .Nm spectrwm
 .Sh DESCRIZIONE
 .Nm
-? un gestore di finestre minimale che cerca di stare in disparte, in modo
-che il prezioso spazio sullo schermo possa essere usato per cose pi?
+è un gestore di finestre minimale che cerca di stare in disparte, in modo
+che il prezioso spazio sullo schermo possa essere usato per cose più
 importanti. Hai dei default sensati e non costringe l'utente ad imparare
-un linguaggio di programmazione per configurarlo. ? stato scritto dagli
+un linguaggio di programmazione per configurarlo. È stato scritto dagli
 hacker per gli hacker e cerca di essere piccolo, compatto e veloce.
 .Pp
 Quando
@@ -35,7 +39,7 @@
 .Pa spectrwm.conf .
 Vedere la sezione
 .Sx FILE DI CONFIGURAZIONE
-pi? sotto.
+più sotto.
 .Pp
 In questa pagina di manuale viene usata la seguente notazione:
 .Pp
@@ -53,21 +57,21 @@
 .El
 .Pp
 .Nm
-? molto semplice da usare. La maggior parte delle operazioni vengono
+è molto semplice da usare. La maggior parte delle operazioni vengono
 svolte usando combinazioni di tasti sulla tastiera o sul mouse.
 Vedere la sezione
 .Sx SCORCIATOIE
-pi? sotto per i default e le possibilit? di personalizzazione.
+più sotto per i default e le possibilità di personalizzazione.
 .Sh FILE DI CONFIGURAZIONE
 .Nm
 prova per prima cosa ad aprire il file di configurazione personale
 dell'utente,
 .Pa ~/.spectrwm.conf .
-Se il file in questione non ? disponibile, prova ad usare il file di
+Se il file in questione non è disponibile, prova ad usare il file di
 configurazione globale
 .Pa /etc/spectrwm.conf .
 .Pp
-Il formato di questo file ? \*(Ltchiave\*(Gt = \*(Ltvalore\*(Gt.
+Il formato di questo file è \*(Ltchiave\*(Gt = \*(Ltvalore\*(Gt.
 Per esempio:
 .Pp
 .Dl color_focus = red
@@ -75,66 +79,211 @@
 Per abilitare o disabilitare un'opzione, usare i valori 1 o 0
 rispettivamente.
 .Pp
+I colori devono essere specificati usando il formato descritto in
+.Xr XQueryColor 3 .
+.Pp
 Il file supporta le seguenti chiavi:
 .Pp
-.Bl -tag -width "title_class_enabledXXX" -offset indent -compact
-.It Cm autorun
+.Bl -tag -width 2m
+.It Ic autorun
 Esegue un'applicazione nel workspace specificato all'avvio.
 Definito nel formato ws[<idx>]:applicazione, eg. ws[2]:xterm esegue un
 xterm nell'area di lavoro 2.
-.It Cm color_focus
-Colore del bordo della finestra che ha il focus.
-.It Cm color_unfocus
-Colore del bordo delle finestre che non hanno il focus.
-.It Cm bar_enabled
-Abilita o disabilita la barra di stato.
-.It Cm bar_border Ns Bq Ar x
+.It Ic bar_action
+Script esterno che aggiunge informazioni come la carica della batteria alla
+barra di stato.
+.It Ic bar_at_bottom
+Posiziona la barra di stato sul fondo dello schermo anziché in cima.
+.It Ic bar_border Ns Bq Ar x
 Colore del bordo della barra di stato nello schermo
 .Ar x .
-.It Cm bar_border_width
+.It Ic bar_border_width
 Spessore del bordo attorno alla barra di stato in pixel. Il bordo
-pu? essere disabilitato usando il valore 0.
-.It Cm bar_color Ns Bq Ar x
+può essere disabilitato usando il valore 0.
+.It Ic bar_color Ns Bq Ar x
 Colore della barra di stato nello schermo
 .Ar x .
-.It Cm bar_font_color Ns Bq Ar x
-Colore del testo della barra di stato nello schermo
-.Ar x .
-.It Cm bar_font
-Font della barra di stato.
-.It Cm bar_action
-Script esterno che aggiunge informazioni come la carica della batteria alla
-barra di stato.
-.It Cm bar_delay
+.It Ic bar_delay
 Frequenza di aggiornamento, in secondi, dello script esterno che aggiunge
 informazioni alla barra di stato.
-.It Cm bar_at_bottom
-Posiziona la barra di stato sul fondo dello schermo anzich? in cima.
-.It Cm stack_enabled
-Abilita o disabilita la visualizzazione dell'algoritmo di stacking
-corrente nella barra di stato.
-.It Cm clock_enabled
+.It Ic bar_enabled
+Abilita o disabilita la barra di stato.
+.It Ic bar_font
+Font usato per la barra di stato. I font possono essere specificati sia nel
+formato Xft che seguendo la X Logical Font Description (XLFD).
+È possibile definire dei font di fallback specificando più font separati da
+virgola. Se tutti i font sono descritti nel formato XLFD, verranno usati dei
+gruppi di font. Se almeno un font è descritto nel formato Xft, verrà usato
+Xft. Se Xft è in uso, viene usato il primo font che è possibile caricare
+correttamente, anche al font mancano dei glifi. Di default vendono usati i
+gruppi di font. Dmenu non supporta Xft.
+.Pp
+Esempi del formato Xft:
+.Bd -literal -offset indent
+bar_font = Terminus:style=Regular:pixelsize=14:antialias=true
+
+bar_font = -*-profont-medium-*-*-*-11-*-*-*-*-*-*-*,Terminus:pixelsize=14,\
+-*-clean-medium-*-*-*-12-*-*-*-*-*-*-*
+.Ed
+.Pp
+Esempi del formato XLFD:
+.Bd -literal -offset indent
+bar_font = -*-terminus-medium-*-*-*-14-*-*-*-*-*-*-*
+
+bar_font = -*-profont-medium-*-*-*-11-*-*-*-*-*-*-*,\
+-*-terminus-medium-*-*-*-14-*-*-*-*-*-*-*,\
+-*-clean-medium-*-*-*-12-*-*-*-*-*-*-*
+.Ed
+.Pp
+Per avere un'elenco dei font disponibile sul sistema usare
+.Xr fc-list 1
+o
+.Xr xlsfonts 1 .
+L'applicazione
+.Xr xfontsel 1
+può essere utile per il formato XLFD.
+.It Ic bar_font_color Ns Bq Ar x
+Colore del testo della barra di stato nello schermo
+.Ar x .
+.It Ic bar_format
+Definisce il formato da utilizzare per il testo visualizzato nella barra di
+stato, sovrascrivendo quanto definito dall'opzione
+.Ic clock_format
+e da tutte le opzioni il cui nome termina con
+.Ic enabled .
+Il formato viene processato con
+.Xr strftime 3
+prima di essere usato.
+Può contenere le seguenti sequenze di caratteri:
+.Bl -column "Sequenza di caratteri" "Sostituita con" -offset indent
+.It Sy "Sequenza di caratteri" Ta Sy "Sostituita con"
+.It Li "+<" Ta "Allinea usando degli spazi"
+.It Li "+A" Ta "Output dello script esterno"
+.It Li "+C" Ta "Classe della finestra"
+.It Li "+D" Ta "Nome dell'area di lavoro"
+.It Li "+F" Ta "Stato floating"
+.It Li "+I" Ta "Indice dell'area di lavoro"
+.It Li "+N" Ta "Numero dello schermo"
+.It Li "+P" Ta "Classe e titolo della finestra, separati da due punti"
+.It Li "+S" Ta "Algoritmo di stacking"
+.It Li "+T" Ta "Titolo della finestra"
+.It Li "+U" Ta "Urgenza"
+.It Li "+V" Ta "Numero di versione"
+.It Li "+W" Ta "Nome della finestra"
+.It Li "++" Ta "Il carattere" Ql +
+.El
+.Pp
+Tutte le sequenze possono limitare la lunghezza del testo generato, usando
+ad esempio +64A. I caratteri che non corrispondono a nessuna sequenza
+vengono lasciati inalterati.
+.It Ic bar_justify
+Giustifica il testo nella barra di stato. I valori possibili sono
+.Pa left
+(sinistra),
+.Pa center
+(centro),
+e
+.Pa right
+(destra).
+.Pp
+Se l'output non è giustificato a sinistra, potrebbe non essere visualizzato
+correttamente in alcune circostanze, a causa degli spazi presenti nel formato
+di default. Vedere l'opzione
+.Ic bar_format
+per maggiori dettagli.
+.It Ic bind Ns Bq Ar x
+Assegna una combinazione di tasti all'azione
+.Ar x .
+Vedere la sezione
+.Sx SCORCIATOIE
+più sotto.
+.It Ic border_width
+Spessore del bordo delle finestre in pixel. Il valore 0 disabilita il bordo.
+.It Ic clock_enabled
 Abilita o disabilita l'orologio nella barra di stato. Se disabilitato,
-usando il valore 0, ? possibile usare un orologio personalizzato nello
+usando il valore 0, è possibile usare un orologio personalizzato nello
 script specificato in
 .Pa bar_action .
-.It Cm dialog_ratio
+.It Ic color_focus
+Colore del bordo della finestra che ha il focus.
+.It Ic color_unfocus
+Colore del bordo delle finestre che non hanno il focus.
+.It Ic dialog_ratio
 Alcune applicazioni hanno finestre di dialogo troppo piccole per risultare
-utili. Questa ? la percentuale dello schermo che verr? usata per le finestre
+utili. Questa è la percentuale dello schermo che verrà usata per le finestre
 di dialogo: ad esempio, 0.6 indica il 60% della dimensione fisica dello
 schermo.
-.It Cm layout
+.It Ic disable_border
+Rimuove il bordo dalle finestre se la barra di stato è nascosta e c'è
+una sola finestra sullo schermo.
+.It Ic focus_close
+Finestra alla quale dare il focus quando la finestra che ha il focus viene
+chiusa. I valori possibili sono
+.Pa first
+(prima),
+.Pa next
+(successiva),
+.Pa previous
+(precedente),
+.Pa last
+(ultima). Il default è
+.Pa previous ;
+.Pa next
+e
+.Pa previous
+sono da intendersi relativi alla finestra appena chiusa.
+.It Ic focus_close_wrap
+Il focus viene passato all'ultima finestra quando la prima viene chiusa, e
+viceversa. Il valore 0 disabilita questo comportamento.
+.It Ic focus_default
+Finestra alla quale dare il focus quando nessuna finestra ha mai avuto il
+focus. I valori possibili sono
+.Pa first
+(prima) e
+.Pa last
+(ultima); il default è
+.Pa last .
+.It Ic focus_mode
+Comportamento del focus in seguito allo spostamento del puntatore del mouse. I
+valori possibili sono:
+.Pp
+.Bl -tag -width "default" -offset indent -compact
+.It Cm default
+Cambia il focus quando il puntatore passa da una finestra all'altra in
+seguito ad uno spostamento o all'interazione con una finestra.
+.It Cm follow
+Cambia il focus ogni volta che il puntatore passa da una finestra all'altra,
+anche quando questo succede in seguito al cambiamento dell'area di lavoro
+o dell'algoritmo di stacking.
+.It Cm manual
+Cambia il focus soltanto in seguito all'interazione con una finestra.
+.El
+.It Ic keyboard_mapping
+Ignora tutte le scorciatoie esistenti e carica delle nuove scorciatoie dal
+file specificato.
+Vedere la sezione
+.Sx FILE DI MAPPATURA DELLA TASTIERA
+per la lista dei file di mappatura che vengono forniti per diversi layout
+di tastiera.
+.It Ic layout
 Layout da utilizzare all'avvio. Definito nel formato
-ws[<idx>]:master_grow:master_add:stack_inc:layout:always_raise:stack_mode,
+ws[idx]:master_grow:master_add:stack_inc:layout:always_raise:stack_mode,
 eg. ws[2]:-4:0:1:0:horizontal assegna il layout orizzontale all'area di
-lavoro 2, riduce l'area principale di 4 unit?, aggiunge una finestra allo
+lavoro 2, riduce l'area principale di 4 unità, aggiunge una finestra allo
 stack e mantiene il comportamento predefinito per quanto riguarda le
 finestre floating.
 I valori possibili per stack_mode sono
-.Pa vertical ,
+.Pa vertical
+(verticale),
+.Pa vertical_flip
+(verticale invertito),
 .Pa horizontal
+(orizzontale),
+.Pa horizontal_flip
+(orizzontale invertito)
 e
-.Pa fullscreen .
+.Pa fullscreen
+(a schermo intero).
 .Pp
 Fare riferimento a
 .Pa master_grow ,
@@ -142,120 +291,116 @@
 .Pa master_add ,
 .Pa master_del ,
 .Pa stack_inc ,
-.Pa stack_del
+.Pa stack_dec
 e
 .Pa always_raise
 per ulteriori informazioni.
-Queste impostazioni sono complesse e hanno effetti secondari; ? opportuno
+Queste impostazioni sono complesse e hanno effetti secondari; è opportuno
 familiarizzare con questi comandi prima di modificare l'opzione
 .Pa layout .
 .Pp
 Questa impostazione non viene applicata dopo il restart.
-.It Cm region
+.It Ic modkey
+Cambia il tasto modificatore.
+Solitamente Mod1 è il tasto ALT e Mod4 è il tasto Windows su un PC.
+.It Ic program Ns Bq Ar p
+Definisce una nuova azione per lanciare il programma
+.Ar p .
+Vedere la sezione
+.Sx PROGRAMMI
+più sotto.
+.It Cm quirk Ns Bq Ar c:n
+Aggiunge un "quirk" (comportamento speciale) per le finestre di classe
+.Ar c
+e nome
+.Ar n .
+Vedere la sezione
+.Sx QUIRK
+più sotto.
+.It Ic region
 Alloca una regione personalizzata, rimuovendo qualsiasi regione
 automaticamente rilevata stia occupando lo stesso spazio sullo schermo.
 Definita nel formato screen[<idx>]:LARGHEZZAxALTEZZA+X+Y, ad esempio
-\& screen[1]:800x1200+0+0.
+screen[1]:800x1200+0+0.
 .Pp
-Per fare s? che pi? monitor vengano considerati come una singola entit?
-? sufficiente creare una regione sufficientemente grande da contenerli,
-eg. screen[1]:2048x760+0+0 unisce due monitor con risoluzione 1024x768
+Per fare sì che più monitor vengano considerati come una singola entità
+è sufficiente creare una regione abbastanza grande da contenerli, ad
+esempio screen[1]:2048x760+0+0 unisce due monitor con risoluzione 1024x768
 posizionati uno di fianco all'altro.
-.It Cm term_width
+.It Ic spawn_position
+Posizione all'interno dello stack per le nuove finestre. I valori possibili
+sono
+.Pa first
+(prima),
+.Pa next
+(successiva),
+.Pa previous
+(precedente) e
+.Pa last
+(ultima). Il default è
+.Pa last ;
+.Pa next
+e
+.Pa previous
+sono relativi alla finestra che ha il focus.
+.It Ic stack_enabled
+Abilita o disabilita la visualizzazione dell'algoritmo di stacking
+corrente nella barra di stato.
+.It Ic term_width
 Imposta la dimensione minima preferita per il terminale. Se questo valore
-? maggiore di 0,
+è maggiore di 0,
 .Nm
-cercher? di riaggiustare la dimensione del testo nel terminale in modo che
+cercherà di adattare la dimensione del testo nel terminale in modo che
 la larghezza del terminale rimanga sopra il valore quando la finestra
 viene ridimensionata. Al momento solo
 .Xr xterm 1
-? supportato. Il binario di
+è supportato. L'eseguibile di
 .Xr xterm 1
-deve essere setuid o setgid perch? questo funzioni: nella maggior parte dei
-sistemi, questo ? il default. L'utente potrebbe voler impostare
+deve essere setuid o setgid perché questo funzioni: nella maggior parte dei
+sistemi, questo avviene di default. L'utente potrebbe voler impostare
 program[term] (vedere la sezione
 .Sx PROGRAMMI
-pi? sotto) per usare una seconda copia del binario di
+più sotto) per usare una seconda copia dell'eseguibile di
 .Xr xterm 1
 che non abbia il bit setgid impostato.
-.It Cm title_class_enabled
+.It Ic title_class_enabled
 Abilita o disabilita la visualizzazione della classe della finestra nella
 barra di stato. Impostare a 1 per abilitare.
-.It Cm title_name_enabled
+.It Ic title_name_enabled
 Abilita o disabilita la visualizzazione del titolo della finestra nella
 barra di stato. Impostare a 1 per abilitare.
-.It Cm urgent_enabled
+.It Ic urgent_enabled
 Abilita o disabilita l'hint "urgente".
 In molti emulatori di terminale, il supporto deve essere abilitato
-separatamente: per xterm, ad esempio, ? necessario aggiungere la riga
+separatamente: per xterm, ad esempio, è necessario aggiungere la riga
 .Pa xterm.urgentOnBell: true
 al file
 .Pa .Xdefaults .
-.It Cm window_name_enabled
-Abilita o disabilita la visualizzazione del nome della finestra nella
-barra di stato. Impostare a 1 per abilitare.
-.It Cm verbose_layout
+.It Ic verbose_layout
 Abilita o disabilita la visualizzazione dei valori correnti di master e
 stack nella barra di stato. Impostare a 1 per abilitare.
-.It Cm modkey
-Cambia il tasto modificatore.
-Solitamente Mod1 ? il tasto ALT e Mod4 ? il tasto Windows su un PC.
-.It Cm focus_mode
-Se viene usato il valore
-.Pa follow_cursor ,
-il gestore di finestre dar? il focus alla finestra sotto il puntatore
-quando si cambia area di lavoro o si creano finestre.
-.It Cm disable_border
-Rimuovi il bordo dalle finestre se la barra di stato ? nascosta e c'?
-una sola finestra sullo schermo.
-.It Cm border_width
-Spessore del bordo delle finestre in pixel. Il valore 0 disabilita il bordo.
-.It Cm program Ns Bq Ar p
-Definisce una nuova azione per lanciare il programma
-.Ar p .
-Vedere la sezione
-.Sx PROGRAMMI
-pi? sotto.
-.It Cm bind Ns Bq Ar x
-Assegna una combinazione di tasti all'azione
-.Ar x .
-Vedere la sezione
-.Sx SCORCIATOIE
-pi? sotto.
-.It Cm quirk Ns Bq Ar c:n
-Aggiunge un "quirk" per le finestre di classe
-.Ar c
-e nome
-.Ar n .
-Vedere la sezione
-.Sx QUIRKS
-pi? sotto.
-.El
-.Pp
-I colori devono essere specificati nel formato usato da
-.Xr XQueryColor 3
-e i font in quello usato da
-.Xr XQueryFont 3 .
+.It Ic window_name_enabled
+Abilita o disabilita la visualizzazione del nome della finestra nella
+barra di stato. Impostare a 1 per abilitare.
 .Pp
-Per avere una lista dei font disponibili sul proprio sistema utilizzare
-.Xr fc-list 1
-o
-.Xr xlsfonts 1 .
-L'applicazione
-.Xr xfontsel 1
-? utile per visualizzare la X Logical Font Description ("XLFD") usata per
-la chiave
-.Pa bar_font .
+Per evitare che un nome particolarmente lungo assegnato ad una finestra
+faccia uscire il testo rimanente dallo schermo, il nome viene troncato a
+64 caratteri. Vedere l'opzione
+.Ic bar_format
+per ulteriori informazioni.
+.It Ic workspace_limit
+Numero di aree di lavoro disponibili. Il minimo è 1, il massimo 22, il
+default 10.
+.El
 .Sh PROGRAMMI
 .Nm
-consente la definizione di azioni personalizzate per lanciare programmi di
-propria scelta, che possono essere assegnate a combinazioni di tasti nello
-stesso modo in cui ? possibile farlo con le azioni predefinite.
-Vedere la sezione
+consente la definizione di azioni personalizzate per lanciare programmi a
+scelta; queste azioni, come quelle predefinite, possono essere assegnate a
+combinazioni di tasti.  Vedere la sezione
 .Sx SCORCIATOIE
-pi? sotto.
+più sotto.
 .Pp
-I programmi di default sono descritte qui sotto:
+I programmi predefiniti sono descritti qui sotto:
 .Pp
 .Bl -tag -width "screenshot_wind" -offset indent -compact
 .It Cm term
@@ -269,20 +414,21 @@
 .It Cm initscr
 initscreen.sh
 .It Cm menu
-dmenu_run \-fn $bar_font \-nb $bar_color \-nf $bar_font_color \-sb $bar_border \-sf $bar_color
+dmenu_run \-fn $bar_font \-nb $bar_color \-nf $bar_font_color \-sb
+$bar_border \-sf $bar_color
 .El
 .Pp
-I programmi personalizzati vengono specificati con la seguente sintassi:
+I programmi personalizzati vengono specificati usando la seguente sintassi:
 .Pp
 .Dl program[<nome>] = <percorso> [<arg> [... <arg>]]
 .Pp
 .Aq nome
-? un qualsiasi identificatore che non va in conflitto con un'azione
+è un qualsiasi identificatore che non va in conflitto con un'azione
 predefinita o una chiave,
 .Aq percorso
-? il programma desiderato, e
+è il programma desiderato, e
 .Aq arg
-sono zero o pi? argomenti da passare al programma.
+sono zero o più argomenti da passare al programma.
 .Pp
 Le seguenti variabili rappresentano valori impostabili in
 .Nm
@@ -304,15 +450,14 @@
 Esempio:
 .Bd -literal -offset indent
 program[ff] = /usr/local/bin/firefox http://spectrwm.org/
-bind[ff] = Mod+f # adesso Mod+F lancia firefox
+bind[ff] = Mod+Shift+b # Adesso Mod+Shift+b lancia Firefox
 .Ed
 .Pp
 Per eliminare la combinazione precedente:
 .Bd -literal -offset indent
-bind[] = Mod+f
+bind[] = Mod+Shift+b
 program[ff] =
 .Ed
-.Pp
 .Sh SCORCIATOIE
 .Nm
 fornisce molte funzioni (o azioni) accessibili tramite combinazioni di
@@ -322,7 +467,7 @@
 .Pp
 .Bl -tag -width "M-j, M-<TAB>XXX" -offset indent -compact
 .It Cm M1
-D? focus alla finestra
+Dà focus alla finestra
 .It Cm M-M1
 Muove la finestra
 .It Cm M-M3
@@ -331,9 +476,9 @@
 Ridimensiona la finestra mantenendola centrata
 .El
 .Pp
-Le scorciatoie da tastiera di default sono:
+Le scorciatoie da tastiera predefinite sono:
 .Pp
-.Bl -tag -width "M-j, M-<TAB>XXX" -offset indent -compact
+.Bl -tag -width "M-j, M-<TAB>XXXXXX" -offset indent -compact
 .It Cm M-S- Ns Aq Cm Return
 term
 .It Cm M-p
@@ -344,8 +489,10 @@
 restart
 .It Cm M- Ns Aq Cm Space
 cycle_layout
+.It Cm M-S-\e
+flip_layout
 .It Cm M-S- Ns Aq Cm Space
-reset_layout
+stack_reset
 .It Cm M-h
 master_shrink
 .It Cm M-l
@@ -357,7 +504,7 @@
 .It Cm M-S-,
 stack_inc
 .It Cm M-S-.
-stack_del
+stack_dec
 .It Cm M- Ns Aq Cm Return
 swap_main
 .It Xo
@@ -382,14 +529,18 @@
 wind_del
 .It Cm M-S-x
 wind_kill
-.It Cm M- Ns Aq Ar n
-.Ns ws_ Ns Ar n
-.It Cm M-S- Ns Aq Ar n
-.Ns mvws_ Ns Ar n
+.It Cm M- Ns Aq Ar 1-9,0,F1-F12
+.Pf ws_ Aq Ar 1-22
+.It Cm M-S- Ns Aq Ar 1-9,0,F1-F12
+.Pf mvws_ Ns Aq Ar 1-22
 .It Cm M- Ns Aq Cm Right
 ws_next
 .It Cm M- Ns Aq Cm Left
 ws_prev
+.It Cm M- Ns Aq Cm Up
+ws_next_all
+.It Cm M- Ns Aq Cm Down
+ws_prev_all
 .It Cm M-a
 ws_prior
 .It Cm M-S- Ns Aq Cm Right
@@ -404,7 +555,7 @@
 version
 .It Cm M-t
 float_toggle
-.It Cm M-S Aq Cm Delete
+.It Cm M-S- Ns Aq Cm Delete
 lock
 .It Cm M-S-i
 initscr
@@ -416,21 +567,43 @@
 always_raise
 .It Cm M-v
 button2
+.It Cm M--
+width_shrink
+.It Cm M-=
+width_grow
+.It Cm M-S--
+height_shrink
+.It Cm M-S-=
+height_grow
+.It Cm M-[
+move_left
+.It Cm M-]
+move_right
+.It Cm M-S-[
+move_up
+.It Cm M-S-]
+move_down
+.It Cm M-S-/
+name_workspace
+.It Cm M-/
+search_workspace
+.It Cm M-f
+search_win
 .El
 .Pp
 I nomi delle azioni e le relative descrizioni sono le seguenti:
 .Pp
-.Bl -tag -width "M-j, M-<TAB>XXX" -offset indent -compact
+.Bl -tag -width "M-j, M-<TAB>XXXX" -offset indent -compact
 .It Cm term
 Lancia un nuovo terminale
-(vedi
+(vedere la sezione
 .Sx PROGRAMMI
-pi? in alto).
+più in alto).
 .It Cm menu
 Menu
-(vedi
+(vedere la sezione
 .Sx PROGRAMMI
-pi? in alto).
+più in alto).
 .It Cm quit
 Chiude
 .Nm .
@@ -439,7 +612,9 @@
 .Nm .
 .It Cm cycle_layout
 Cambia layout.
-.It Cm reset_layout
+.It Cm flip_layout
+Inverte l'area principale e l'area di stacking.
+.It Cm stack_reset
 Re-inizializza il layout.
 .It Cm master_shrink
 Restringe l'area principale.
@@ -450,23 +625,23 @@
 .It Cm master_del
 Rimuove finestre dall'area principale.
 .It Cm stack_inc
-Aggiunge righe/colonne all'area di stacking.
-.It Cm stack_del
-Rimuove righe/colonne dall'area di stacking.
+Aumenta il numero di righe/colonne dell'area di stacking.
+.It Cm stack_dec
+Riduce il numero di righe/colonne dell'area di stacking.
 .It Cm swap_main
 Muove la finestra corrente nell'area principale.
 .It Cm focus_next
-D? il focus alla finestra successiva.
+Dà il focus alla finestra successiva.
 .It Cm focus_prev
-D? il focus alla finestra precedente.
+Dà il focus alla finestra precedente.
 .It Cm focus_main
-D? il focus alla finestra principale.
+Dà il focus alla finestra principale.
 .It Cm swap_next
 Scambia con la finestra successiva dell'area di lavoro.
 .It Cm swap_prev
 Scambia con la finestra precedente dell'area di lavoro.
 .It Cm bar_toggle
-Mostra/nascondi la barra di stato da tutte le aree di lavoro.
+Mostra/nascondi la barra di stato.
 .It Cm wind_del
 Chiude la finestra corrente.
 .It Cm wind_kill
@@ -476,17 +651,21 @@
 .Ar n ,
 dove
 .Ar n
-? compreso tra 1 e 10.
+è compreso tra 1 e workspace_limit.
 .It Cm mvws_ Ns Ar n
 Sposta la finestra corrente nell'area di lavoro
 .Ar n ,
 dove
 .Ar n
-? compreso tra 1 e 10.
+è compreso tra 1 e workspace_limit.
 .It Cm ws_next
 Passa all'area di lavoro non vuota successiva.
 .It Cm ws_prev
 Passa all'area di lavoro non vuota precedente.
+.It Cm ws_next_all
+Passa all'area di lavoro successiva.
+.It Cm ws_prev_all
+Passa all'area di lavoro precedente.
 .It Cm ws_prior
 Passa all'ultima area di lavoro visitata.
 .It Cm screen_next
@@ -496,12 +675,12 @@
 .It Cm screenshot_all
 Cattura uno screenshot dell'intero schermo, se abilitato (vedere la sezione
 .Sx PROGRAMMI
-pi? in alto).
+più in alto).
 .It Cm screenshot_wind
 Cattura uno screenshot di una singola finestra, se abilitato (vedere la
 sezione
 .Sx PROGRAMMI
-pi? in alto).
+più in alto).
 .It Cm version
 Abilita/disabilita il numero di versione nella barra di stato.
 .It Cm float_toggle
@@ -509,20 +688,42 @@
 .It Cm lock
 Blocca lo schermo (vedere la sezione
 .Sx PROGRAMMI
-pi? in alto).
+più in alto).
 .It Cm initscr
 Re-inizializza gli schermi fisici (vedere la sezione
 .Sx PROGRAMMI
-pi? in alto).
+più in alto).
 .It Cm iconify
 Minimizza (unmap) la finesta che ha il focus.
 .It Cm uniconify
 Massimizza (map) la finestra selezionata tramite dmenu.
 .It Cm always_raise
-Quando ? abilitato, le finestre floating possono essere oscurate da
+Quando è abilitato, le finestre floating possono essere oscurate da
 finestre tiled.
 .It Cm button2
 Simula la pressione del tasto centrale del mouse.
+.It Cm width_shrink
+Riduce la larghezza di una finestra floating.
+.It Cm width_grow
+Aumenta la larghezza di una finestra floating.
+.It Cm height_shrink
+Riduce l'altezza di una finestra floating.
+.It Cm height_grow
+Aumenta l'altezza di una finestra floating.
+.It Cm move_left
+Sposta una finestra floating verso sinistra.
+.It Cm move_right
+Sposta una finestra floating verso destra.
+.It Cm move_up
+Sposta una finestra floating verso l'alto.
+.It Cm move_down
+Sposta una finestra floating verso il basso.
+.It Cm name_workspace
+Assegna un nome all'area di lavoro corrente.
+.It Cm search_workspace
+Cerca un'area di lavoro.
+.It Cm search_win
+Cerca una finestra.
 .El
 .Pp
 Le scorciatoie personalizzate sono specificate nel file di configurazione
@@ -531,9 +732,9 @@
 .Dl bind[<azione>] = <tasti>
 .Pp
 .Aq azione
-? una delle azioni elencate sopra (oppure nulla) e
+è una delle azioni elencate sopra (oppure nulla) e
 .Aq tasti
-? dato da zero o pi? modificatori (MOD, Mod1, Shift, ecc.) e uno o pi?
+è dato da zero o più modificatori (MOD, Mod1, Shift, ecc.) e uno o più
 tasti normali (b, space, ecc.), separati da "+".
 Per esempio:
 .Bd -literal -offset indent
@@ -541,17 +742,66 @@
 bind[] = Mod1+q # rimuovi l'assegnazione di Alt + q
 .Ed
 .Pp
-Pi? combinazioni di tasti possono essere assegnate alla stessa azione.
+Per usare il valore corrente di
+.Ic
+modkey ,
+specificare MOD come modificatore.
+.Pp
+Più combinazioni di tasti possono essere assegnate alla stessa azione.
+.Pp
+Per utilizzare caratteri come å e π in una scorciatoia è necessario
+specificare il nome xkb del carattere, non il carattere stesso. Il nome
+del carattere può essere trovato lanciando
+.Xr xev 1 ,
+dando focus alla finestra, premendo il tasto in questione e leggendo
+l'output su terminale. Ad esempio, per il carattere å:
+.Bd -literal -offset indent
+KeyPress event, serial 41, synthetic NO, window 0x2600001,
+    root 0x15a, subw 0x0, time 106213808, (11,5), root:(359,823),
+    state 0x0, keycode 24 (keysym 0xe5, aring), same_screen YES,
+    XLookupString gives 2 bytes: (c3 a5) "å"
+    XmbLookupString gives 2 bytes: (c3 a5) "å"
+    XFilterEvent returns: False
+.Ed
+.Pp
+In questo caso, il nome del carattere è aring: nel file di configurazione
+si dovrà quindi scrivere
+.Bd -literal -offset indent
+bind[program] = MOD+aring
+.Ed
+.Sh FILE DI MAPPATURA DELLA TASTIERA
+I file di mappatura per svariati layout di tastiera sono elencati qui
+sotto. Questi file possono essere utilizzati tramite la chiave di
+configurazione
+.Pa keyboard_mapping
+per caricare scorciatoie create appositamente per un determinato layout
+di tastiera.
+.Pp
+.Bl -tag -width "spectrwm_XX.confXXX" -offset indent -compact
+.It Cm spectrwm_cz.conf
+Layout di tastiera per la Repubblica Ceca
+.It Cm spectrwm_es.conf
+Layout di tastiera per la Spagna
+.It Cm spectrwm_fr.conf
+Layout di tastiera per la Francia
+.It Cm spectrwm_fr_ch.conf
+Layout di tastiera per la Svizzera Francese
+.It Cm spectrwm_se.conf
+Layout di tastiera per la Svezia
+.It Cm spectrwm_us.conf
+Layout di tastiera per gli Stati Uniti
+.El
 .Sh QUIRK
 .Nm
-fornisce la possibilit? di specificare dei "quirk" per la gestione di
+fornisce la possibilità di specificare dei "quirk" per la gestione di
 finestre che devono subire un trattamento speciale da un gestore di finestre
 tiling, come ad esempio alcune finestre di dialogo e applicazioni a
 schermo intero.
 .Pp
 I quirk abilitati di default sono elencati qui sotto:
 .Pp
-.Bl -tag -width "OpenOffice.org N.M:VCLSalFrame<TAB>XXX" -offset indent -compact
+.Bl -tag -width "OpenOffice.org N.M:VCLSalFrame<TAB>XXX" -offset indent \
+-compact
 .It Firefox\-bin:firefox\-bin
 TRANSSZ
 .It Firefox:Dialog
@@ -597,8 +847,8 @@
 Rimuove i bordi, consentendo alla finestra di usare l'intera dimensione
 dello schermo.
 .It FOCUSPREV
-Quando la finestra viene chiusa, d? il focus alla finestra che aveva il
-focus precedente anzich? all'applicazione precedente nello stack.
+Quando la finestra viene chiusa, dà il focus alla finestra che aveva il
+focus precedentemente anziché all'applicazione precedente nello stack.
 .El
 .Pp
 I quirk personalizzati vanno specificati nel file di configurazione come
@@ -611,35 +861,37 @@
 .Aq nome
 specificano a quali finestre i quirk vanno applicati, e
 .Aq quirk
-? uno dei quirk presente nella lista sopra.
+è uno dei quirk presente nella lista sopra.
 Ad esempio:
 .Bd -literal -offset indent
 quirk[MPlayer:xv] = FLOAT + FULLSCREEN + FOCUSPREV
 quirk[pcb:pcb] = NONE  # rimuovi un quirk precedentemente specificato
 .Ed
 .Pp
-? possibile ottenere
+È possibile ottenere
 .Aq classe
 e
 .Aq nome
-usando xprop(1) e facendo click sulla finestra desiderata.
-Nel seguente esempio ? stato fatto click sulla finestra principale di
-Firefox:
+usando
+.Xr xprop 1
+e facendo click sulla finestra desiderata. Nel seguente esempio è stato
+fatto click sulla finestra principale di Firefox:
 .Bd -literal -offset indent
 $ xprop | grep WM_CLASS
 WM_CLASS(STRING) = "Navigator", "Firefox"
 .Ed
 .Pp
-Bisogna tenere conto del fatto che usare grep per trovare WM_CLASS inverte
-la classe e il nome. Nell'esempio precedente, la dichiarazione del quirk
-sarebbe
+Bisogna tenere conto del fatto che usare
+.Xr grep 1
+per trovare WM_CLASS inverte la classe e il nome. Nell'esempio precedente,
+la dichiarazione del quirk sarebbe
 .Bd -literal -offset indent
 quirk[Firefox:Navigator] = FLOAT
 .Ed
 .Pp
 .Nm
 assegna automaticamente i quirk alle finestre in base al valore della
-propriet? _NET_WM_WINDOW_TYPE in base al seguente schema:
+proprietà _NET_WM_WINDOW_TYPE in base al seguente schema:
 .Pp
 .Bl -tag -width "_NET_WM_WINDOW_TYPE_TOOLBAR<TAB>XXX" -offset indent -compact
 .It _NET_WM_WINDOW_TYPE_DOCK
@@ -654,25 +906,25 @@
 FLOAT
 .El
 .Pp
-In tutti gli altri casi, nessun quirk ? automaticamente assegnato alla
-finestra. I quirk specificati nel file di configurazione hanno la precedenza
-sui quirk assegnati in automatico.
+In tutti gli altri casi, alle finestre non viene assegnato alcun quirk. I
+quirk specificati nel file di configurazione hanno la precedenza sui
+quirk assegnati in automatico.
 .Sh EWMH
 .Nm
 implementa in maniera parziale la specifica Extended Window Manager Hints
-(EWMH). Ci? permette di controllare sia le finestre che
+(EWMH). Questo permette di controllare sia le finestre che
 .Nm
-stesso tramite script e programmi esterni. Per renderlo possibile,
+stesso tramite script e programmi esterni. Per rendere ciò possibile,
 .Nm
 risponde ad alcuni eventi di tipo ClientMessage; questo tipo di messaggio
-pu? essere inviato da un terminale usando programmi come
+può essere inviato da un terminale usando programmi come
 .Xr wmctrl 1
 e
 .Xr xdotool 1 .
 Per il formato esatto di questi messaggi, si veda la specifica EWMH.
 .Pp
-L'id della finestra che ha il focus ? memorizzato nella propriet?
-_NET_ACTIVE_WINDOW della root window. ? quindi possibile ottenere il titolo
+L'id della finestra che ha il focus è memorizzato nella proprietà
+_NET_ACTIVE_WINDOW della root window. È quindi possibile ottenere il titolo
 della finestra attiva usando
 .Xr xprop 1
 e
@@ -682,7 +934,7 @@
 $ xprop \-id $WINDOWID WM_NAME | grep \-o "\\".*\\""
 .Ed
 .Pp
-Per dare il focus ad una finestra, ? sufficiente inviare il messaggio
+Per dare il focus ad una finestra, è sufficiente inviare il messaggio
 _NET_ACTIVE_WINDOW alla root window. Ad esempio, usando
 .Xr wmctrl 1
 (supponendo che 0x4a0000b sia l'id della finestra a cui dare il focus):
@@ -690,7 +942,7 @@
 $ wmctrl \-i \-c 0x4a0000b
 .Ed
 .Pp
-Per chiudere una finestra si pu? inviare il messaggio _NET_CLOSE_WINDOW
+Per chiudere una finestra si può inviare il messaggio _NET_CLOSE_WINDOW
 alla root window. Ad esempio, usando
 .Xr wmctrl 1
 (supponendo che 0x4a0000b sia l'id della finestra da chiudere):
@@ -698,8 +950,8 @@
 $ wmctrl \-i \-c 0x4a0000b
 .Ed
 .Pp
-Per passare una finestra da floating a tiled si pu? aggiungere o rimuovere
-l'atomo _NET_WM_STATE_ABOVE alla propriet? _NET_WM_STATE della finestra,
+Per passare una finestra da floating a tiled si può aggiungere o rimuovere
+l'atomo _NET_WM_STATE_ABOVE alla proprietà _NET_WM_STATE della finestra,
 inviando il messaggio _NET_WM_STATE alla root window. Ad esempio, usando
 .Xr wmctrl 1
 (supponendo che 0x4a0000b sia l'id della finestra):
@@ -719,7 +971,7 @@
 .Pp
 I messaggi _NET_MOVERESIZE_WINDOW vengono ignorati per le finestre stacked.
 .Sh SEGNALI
-? possibile riavviare
+È possibile riavviare
 .Nm
 inviandogli il segnale HUP.
 .Sh FILE
@@ -739,7 +991,7 @@
 .An -nosplit
 .Pp
 .Nm
-? stato scritto da:
+è stato scritto da:
 .Pp
 .Bl -tag -width "Ryan Thomas McBride Aq mcbride@countersiege.com " -offset indent -compact
 .It Cm Marco Peereboom Aq marco@peereboom.us
@@ -748,8 +1000,8 @@
 .It Cm Pierre-Yves Ritschard Aq pyr@spootnik.org
 .It Cm Tuukka Kataja Aq stuge@xor.fi
 .It Cm Jason L. Wright Aq jason@thought.net
+.It Cm Reginald Kennedy Aq rk@rejii.com
+.It Cm Lawrence Teo Aq lteo@lteo.net
+.It Cm Tiago Cunha Aq tcunha@gmx.com
+.It Cm David Hill Aq dhill@mindcry.org
 .El
-.Sh BUGS
-Al momento il menu, invocato usando
-.Cm M-p ,
-dipende da dmenu.
